markdown/komand_markdown/actions/markdown_to_txt/action.py

```diff
@@ -0,0 +1,43 @@
+import komand
+from .schema import MarkdownToTxtInput, MarkdownToTxtOutput, Input, Output, Component
+from komand_markdown.util import utils
+# Custom imports below
+from bs4 import BeautifulSoup
+import binascii
+
+
+class MarkdownToTxt(komand.Action):
+
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name='markdown_to_txt',
+            description=Component.DESCRIPTION,
+            input=MarkdownToTxtInput(),
+            output=MarkdownToTxtOutput())
+
+    def run(self, params={}):
+        markdown_bytes = params.get(Input.MARKDOWN)
+        markdown_string = params.get(Input.MARKDOWN_STRING)
+
+        if not (((markdown_string is None) ^ (markdown_bytes is None))
+                or ((markdown_string == "") ^ (markdown_bytes == ""))):
+            raise Exception("You must define one of Markdown or Markdown String")
```

Comment:
Should raise a PluginException here https://komand.github.io/python/error_handling.html#plugin-exceptions

Reply:
We tried add here PluginException, but this plugin is in `komand/python-pypy3-plugin:2`. This version of image also have `PluginException`? If yes could you tell us what is an import? We tried this: `from komand.exceptions import PluginException` and it failed. We can try this again.

Reply:
Ah, it's using an older version of SDK runtime. Never mind then.